### PR TITLE
Update cardano-graphql and associated types

### PR DESCRIPTION
**Pull request by Sam-Jeston**
We should merge the S3 branch before this one, so that this gives us a working, deployed app

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,21 +1,21 @@
 version: '3.7'
 services:
   api:
-    image: samjeston/cardano-graphql-dev:0.1.6
+    image: samjeston/cardano-graphql-dev:1096ec1
```
Comment (rhyslbw) [inline docker-compose.yml]: Should this be?
```suggestion
    image: samjeston/cardano-graphql-dev:0.2.1
```

Comment (Sam-Jeston) [reply docker-compose.yml]: Yep, done

File: source/features/blocks/api/transformers.ts [hunk 2]
```diff
@@ -13,21 +13,29 @@ export const blockInfoTransformer = (b: BlockInfoFragment): IBlockInfo => ({
...
-): IBlockOverview => ({
-  ...blockInfoTransformer(b),
-  createdAt: 1568366883000, // TODO: missing API data
-  createdBy: 'af2800c', // TODO: missing API data
-  epoch: b.epoch ? b.epoch.number : 0,
-  output: 11189.647356, // TODO: missing API data
-  transactions: b.transactions ? b.transactions.length : 0,
-});
+): IBlockOverview => {
+  const transactions =
```
Comment (rhyslbw) [inline source/features/blocks/api/transformers.ts]: This guarding won't be required in the next version of Cardano GraphQL

File: source/features/blocks/api/transformers.ts [hunk 3]
```diff
@@ -13,21 +13,29 @@ export const blockInfoTransformer = (b: BlockInfoFragment): IBlockInfo => ({
...
+    b.transactions_aggregate &&
+    b.transactions_aggregate.aggregate &&
+    b.transactions_aggregate.aggregate.count
+      ? b.transactions_aggregate.aggregate.count
+      : 0;
+  return {
+    ...blockInfoTransformer(b),
+    createdAt: b.createdAt,
+    createdBy: 'af2800c', // TODO: missing API data
```
Comment (rhyslbw) [inline source/features/blocks/api/transformers.ts]: Coming in next version

**Pull request merged**